src/components/report/candidate.js

```diff
@@ -1,14 +1,19 @@
 import {useEffect} from "react";
 import {useRecoilState} from "recoil";
+import Feedback from "components/results/feedback";
 import ArchetypeHeading from "components/results/personality/archetype/heading";
 import ArchetypeTips from "components/results/personality/archetype/tips";
 import Dimensions from "components/results/personality/dimension/list";
 import Traits from "components/results/personality/trait/list";
+import useAssessment from "lib/hooks/use-assessment";
 import {optionsState} from "lib/recoil";
 import style from "./style.scss";
 
 export default function CandidateReport() {
   const [options, setOptions] = useRecoilState(optionsState);
+  const assessment = useAssessment();
+
+  const showFeedback = assessment?.deck_id.includes("big-five");
```

Comment:
If assessment is null, won't this still error for doing `null.includes`? I think maybe a second `?` is necessary, but I haven't looked into `?` too much

Reply:
![image](https://github.com/traitify/traitify-widgets/assets/30631476/a39fdf55-b966-4d31-9a81-867c1711de81)
![image](https://github.com/traitify/traitify-widgets/assets/30631476/8957086c-e9d3-4a41-9a9b-efa1b9c6fb4e)


`showFeedback` would be `undefined` - falsy